02_configure_host.sh

```diff
@@ -42,7 +42,7 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -e "virthost=$HOSTNAME" \
     -e "vm_platform=$NODES_PLATFORM" \
     -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
-    -e "provisioning_url_host=$PROVISIONING_URL_HOST" \
+    -e "provisioning_url_host=$PROVISIONING_HOST_IP" \
```

Comment:
Why'd this change? `$PROVISIONING_URL_HOST` holds the bracketed host portion if it's IPv6

Reply:
`$PROVISIONING_URL_HOST` doesn't exist at all

Reply:
Oh, well that's a good reason for changing it.

I don't really think we use it in dev-scripts anymore anyway, since it's used to set deploy_kernel/ramdisk in the nodes json -- but these days all this is calculated in the installer.

But just so this is correct, could you use `$(wrap_if_ipv6 PROVISIONING_HOST_IP)`?